manifests/a/Audinate/DanteVirtualSoundcard/4.4.1.3/Audinate.DanteVirtualSoundcard.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Audinate.DanteVirtualSoundcard
+PackageVersion: 4.4.1.3
+InstallerType: burn
+Installers:
+- InstallerUrl: https://audinate-software-updates.sgp1.cdn.digitaloceanspaces.com/DanteVirtualSoundcard/4/4.4/DVS-4.4.1.3_windows.exe
+  Architecture: x64
+  InstallerSha256: 9AAA0563D93FDF58EF76EA19A58F61F80F5E926908ADE7FBFC85BD41B4D44707
```

```suggestion
  InstallerSha256: 9AAA0563D93FDF58EF76EA19A58F61F80F5E926908ADE7FBFC85BD41B4D44707
  InstallerSuccessCodes: 3010
```

I'm not completely sure, but I believe something like this might allow the PR to merge. [Schema](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.6.0/installer.md), for reference.

---

This is expecting an array. > fixed it in my updated commit.